### PR TITLE
BWDB-4711

### PR DESCRIPTION
**Pull request by antkoval**
## PR Checklist

Check these before submitting your pull request:

- [ ] Visual and behavioral components are separated
- [ ] Exported components are documented with examples
- [ ] Props have JSDoc comments
- [ ] All relevant visual sub-components can be overridden via props
- [ ] Any extra props are spread into the outermost rendered element

## Breaking Changes

* Describe all breaking changes here

## Changes to Existing Components

* Describe meaningful changes to existing components here

## New Visual Components

* List and describe any new visual components (styled-component primitives) here. Visual components which are sub-components of exported components must be contained in a `/styles` directory adjacent to their super-component

## New Behavioral Components

* List and describe behavioral components here. Behavioral components must have no associated styling. If styling is necessary, refactor styled elements into visual components and ensure they can be overridden via props.

### CODE REVIEW

File: src/components/Button/SubmitButton.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ class SubmitButton extends React.Component {
       return <span>{pristineContents}</span>;
     }
     if (loading) {
-      return <Loader size="14px" />;
```
Comment (tedwards-bw) [inline src/components/Button/SubmitButton.js]: On my display, setting Loader size to 16px does not keep the height constant between the states (Pristine, Loading, etc...).  Also, do we want the Ring size to increase, or do we want the overall button height to be constant with contents centered vertically? You might want to check w/ other Forge team members about the best way to fix it.

Comment (zelitzsi) [reply src/components/Button/SubmitButton.js]: @tedwards-bw The requirements should come from UX.

Comment (antkoval) [reply src/components/Button/SubmitButton.js]: changed the loader size back and adjusted buttons min-height

Comment (tedwards-bw) [reply src/components/Button/SubmitButton.js]: Should the min-height be confined to Button.Submit (the only thing that replaces its contents with a Loader)?  Would Button.Small, for example, be impacted by this?

Comment (antkoval) [reply src/components/Button/SubmitButton.js]: I changed the solution to affect only submit button

**Pull request merged**